build/common.sh

```diff
@@ -714,6 +714,7 @@ function kube::build::copy_output() {
     --filter='+ /vendor/' \
     --filter='+ /staging/***/Godeps/**' \
     --filter='+ /_output/dockerized/bin/**' \
+    --filter='- /_output/dockerized/go/**' \
```

Comment:
Hmm, I guess this works. At some point I hope to eliminate the rsync and use a bind mount so we'll have to deal with this later. IMHO we should fix the permissions issue instead. https://github.com/kubernetes/kubernetes/issues/112862